lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb

```diff
@@ -177,35 +177,6 @@ def generate_settings_to_import_pod_targets
           end
         end
 
-        # Ensure to add the default linker run path search paths as they could
```

Comment:
This method was just moved to `xcconfig_helper` since now multiple callers can use it. No changes in its logic.